.changeset/warm-points-wonder.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+bugfix: missing compatibility_date
+Fixed the compatibility_date not being in the wrangler.toml if one wasn't upstream in the script metadata.
```

Comment:
We should be writing these for the benefit of folks who have no context what this is
```suggestion
Fixed a bug where compatibility_date wasn't being added to wrangler.toml when initializing a worker via `wrangler init --from-dash`
```

Reply:
Yeah I attempt to.